src/d3a/models/area/__init__.py

```diff
@@ -618,3 +620,19 @@ def update_config(self, **kwargs):
             self.strategy.read_config_event()
         for child in self.children:
             child.update_config(**kwargs)
+
+    def is_market_spot(self, market_id: str) -> bool:
```

Comment:
Yeah, I was refactoring these a lot forth and back. If you have a look at the caller of this method [here](https://github.com/gridsingularity/d3a/pull/1261/files/7f50801cca399e6c2d92e261de02a41ab571a25e#diff-f66774415440ea8a12f186f18c94dc58a2175b62ed94c9ab09ce8298b3eef863R179) would agree that `self.area.is_market_spot` is less confusing than `self.area.is_spot_market` ? Because in the latter version you could read "Area is spot market" which does not make sense IMO.
Maybe you or others have a better proposal to name this properly?

Reply:
It really seems off to have this attribute/property in the Area class, the two names don't make such sense

I would vote for getting rid of these functions and only have 1 method that returns the type of market (spot, future, settlement ...)